backend/src/services/tool.py

```
@@ -91,18 +91,21 @@ def agent_cards(a2a: dict[str, A2AServer]):
-    #     tool: StructuredTool = next(
-    #         (tool for tool in TOOL_LIBRARY if tool.name == name), None
-    #     )
-    #     if not tool:
-    #         raise ValueError(f"Tool {name} not found")
-    #     return await tool.ainvoke(
-    #         input=input,
-    #         config={"configurable": {"user_id": self.user_id, **config}}
-    #         if self.user_id
-    #         else None,
-    #     )
+    async def invoke_default_tool(self, name: str, input: dict, config: dict = None):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)